convince-my-boss.html

```diff
@@ -46,9 +46,11 @@
     <!-- <div id="preloader">
     <div id="load"></div>
   </div> -->
-    <nav class="navbar navbar-custom navbar-fixed-top" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
+    <nav class="navbar navbar-custom navbar-fixed-top top-nav-collapse" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
         <div class="container">
-            <img src="img/banner-logo.png" class="logo" alt="banner-logo" id="logo" style="width: 150px;">
+            <a class="navbar-brand" rel="home" href="https://in.pycon.org/2018/" title="PyCon India 2018">
```

Comment:
Ok this looks like a bug. This has to be done in all pages. Can you open a separate issue and solve this in a separate PR ? Also the background color doesn't look good with the logo so just use white background.

Reply:
Yeah, sure. Please review issue #135